src/components/CounterPage/CounterContainer/index.js

```diff
@@ -0,0 +1,58 @@
+import { Component } from "react";
+import Counter from "../Counter";
+
+
+class CounterContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      countValue: 0,
+      isEven: true
+    }
+  }
+
+  handleIncrease = () => {
+    this.setState({countValue: this.state.countValue + 1});
```

Comment:
As I told, better to update state with callback function inside `setState` method instead of just passing object. 
Rewrite it here and in code below